diff --git a/mautic/plugins/MauticCrmBundle/Integration/HubspotIntegration.php b/mautic/plugins/MauticCrmBundle/Integration/HubspotIntegration.php
index 8444b44f..26176088 100644
--- a/mautic/plugins/MauticCrmBundle/Integration/HubspotIntegration.php
+++ b/mautic/plugins/MauticCrmBundle/Integration/HubspotIntegration.php
@@ -162,7 +162,7 @@ class HubspotIntegration extends CrmAbstractIntegration
      */
     public function getAvailableLeadFields($settings = [])
     {
-        if ($fields = parent::getAvailableLeadFields()) {
+        if ($fields = parent::getAvailableLeadFields($settings)) {
             return $fields;
         }
 
